languages.toml

```diff
@@ -2279,6 +2282,20 @@ file-types = ["bzl", "bazel", "star", { glob = "BUILD" }, { glob = "BUILD.*" },
 comment-token = "#"
 indent = { tab-width = 4, unit = "    " }
 grammar = "python"
+language-servers = ['starlark-lsp']
+
+
+[[language]]
+name = "buck2"
```

Comment:
I believe this should be merged into the starlark definition instead. We don't have a `bazel` language for example, just starlark

Reply:
Fair enough, we should probs ads the .buckconfig files to ini aswell then.

Reply:
Is there a syntax for using the `buck2 lsp` as opposed to `starlark-lsp` if we do it that way? {they are _not_ the same}

Reply:
You can have `language-servers = ["starlark-lsp", "buck2"]` and whichever is installed will be used

Reply:
> You can have `language-servers = ["starlark-lsp", "buck2"]` and whichever is installed will be used

Sorry, I was not clear.
You would _NOT_ want the buck2 one to fire up when writing starlark outside of a buck context, the regular one would be preferable no?

Reply:
For that case I think we can configure the buck language server with a required root pattern so that it is not started unless a buck-specific file is present in the workspace